eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/model/eva/SubmittedVariantOperationEntity.java

```diff
@@ -33,6 +34,27 @@ public SubmittedVariantOperationEntity() {
         setCreatedDate(LocalDateTime.now());
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        SubmittedVariantOperationEntity that = (SubmittedVariantOperationEntity) o;
+        return Objects.equals(this.getId(), that.getId()) &&
+                Objects.equals(this.getEventType(), that.getEventType()) &&
+                Objects.equals(this.getAccession(), that.getAccession()) &&
+                Objects.equals(this.getMergedInto(), that.getMergedInto()) &&
+                Objects.equals(this.getSplitInto(), that.getSplitInto()) &&
+                Objects.equals(this.getReason(), that.getReason()) &&
+                Objects.equals(this.getCreatedDate(), that.getCreatedDate()) &&
+                Objects.equals(this.getInactiveObjects(), that.getInactiveObjects());
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(this.getId(), this.getEventType(), this.getAccession(), this.getMergedInto(),
+                            this.getSplitInto(),this.getReason(), this.getCreatedDate(), this.getInactiveObjects());
```

Comment:
Hopefully these are resilient with null values...

Reply:
According to the documentation it should work fine with nulls.